### ISSUE TITLE
New Valheim version Hearth & Home released (september 16th) seems to be incompatible or having trouble with BepInEx

### ISSUE THREAD

**Issue by ppeau** [BepInEx/BepInEx#302]
### What happened?

New Valheim version Hearth & Home released today september 16th seems to be incompatible or having trouble with BepInEx

During load bepinex:

![image](https://user-images.githubusercontent.com/357328/133663453-455d8a37-e8aa-4bec-b645-76c6c5bdd7b4.png)

### BepInEx Distribution

Stable from GitHub

### Log outputs

```txt
see above

from : https://valheim.thunderstore.io/package/denikson/BepInExPack_Valheim/
```

### Environment

```markdown
- OS: windows 10
- BepInEx: 5.4.15.0
- Game: Valheim
```

**Comment by anacedragon** [comment]
same issue here, using BepInEx 5.4.15.0 (installed via r2modman) for Valheim Patch 0.202.14 on Windows 10. all mods disabled except BepInEx and BepInEx Configuration Manager. (I don't often use GitHub, so if any additional information is needed, let me know.)

> [Message:   BepInEx] BepInEx 5.4.15.0 - valheim (7/2/2021 3:55:26 PM)
[Info   :   BepInEx] Running under Unity v2019.4.24.6137584
[Info   :   BepInEx] CLR runtime version: 4.0.30319.42000
[Info   :   BepInEx] Supports SRE: True
[Info   :   BepInEx] System platform: Bits64, Windows
[Message:   BepInEx] Preloader started
[Info   :   BepInEx] Loaded 1 patcher method from [BepInEx.Preloader 5.4.15.0]
[Info   :   BepInEx] 1 patcher plugin loaded
[Info   :   BepInEx] Patching [UnityEngine.CoreModule] with [BepInEx.Chainloader]
[Message:   BepInEx] Preloader finished
[Message:   BepInEx] Chainloader ready
[Message:   BepInEx] Chainloader started
[Info   :   BepInEx] 2 plugins to load
[Info   :   BepInEx] Loading [Configuration Manager 16.1]
[Info   :   BepInEx] Loading [Display BepInEx Info In-Game 1.1.0]
[Warning:  HarmonyX] AccessTools.DeclaredMethod: Could not find method for type Console and name UpdateChat and parameters
[Error  : Unity Log] ArgumentException: Undefined target method for patch method static void Valheim.DisplayBepInExInfo.DisplayInfoPlugin::FixConsoleMesh2()
Stack trace:
HarmonyLib.PatchClassProcessor.PatchWithAttributes (System.Reflection.MethodBase& lastOriginal) (at <48ac0133328b486983afe98eee5b730e>:0)
HarmonyLib.PatchClassProcessor.Patch () (at <48ac0133328b486983afe98eee5b730e>:0)
Rethrow as HarmonyException: Patching exception in method null
HarmonyLib.PatchClassProcessor.ReportException (System.Exception exception, System.Reflection.MethodBase original) (at <48ac0133328b486983afe98eee5b730e>:0)
HarmonyLib.PatchClassProcessor.Patch () (at <48ac0133328b486983afe98eee5b730e>:0)
HarmonyLib.Harmony.PatchAll (System.Type type) (at <48ac0133328b486983afe98eee5b730e>:0)
HarmonyLib.Harmony.CreateAndPatchAll (System.Type type, System.String harmonyInstanceId) (at <48ac0133328b486983afe98eee5b730e>:0)
Valheim.DisplayBepInExInfo.DisplayInfoPlugin.Awake () (at <1b9228d9ccf74a1c993491a05ff92b6f>:0)
UnityEngine.GameObject:AddComponent(Type)
BepInEx.Bootstrap.Chainloader:Start()
UnityEngine.Application:.cctor()
> 
> [Message:   BepInEx] Chainloader startup complete
[Info   : Unity Log] 09/16/2021 15:13:47: Starting to load scene:start

**Comment by ghorsington** [comment]
Greetings!

As the errors imply, this is an issue with [Valheim.DisplayBepInExInfo](https://github.com/Valheim-Modding/Valheim.DisplayBepInExInfo) plugin which is included with the Valheim BepInExPack on Thunderstore. Stock BepInEx works fine as before.

The [plugin](https://github.com/Valheim-Modding/Valheim.DisplayBepInExInfo/releases/tag/v2.0.0) was just updated with the necessary support and the [pack](https://valheim.thunderstore.io/package/denikson/BepInExPack_Valheim/5.4.1501/) now includes the updated plugin. As such, I'll mark this as resolved.

**Comment by Phayntom** [comment]
I am having the exact same problem even after installing new mentioned BepInEx 5.4.1501 and also updating the  Valheim.DisplayBepInExInfo plugin to version 2.0.0 as shown above.  

Just to be sure, I uninstalled game.  Re-installed game.  Opened Valheim.  Closed Valheim  Then copied over my backup plugin and config folders, and also restored my saved games folder located in C:\Users\<user>\AppData\LocalLow\IronGate.

I got the game and BepInEx to load technically, but even after restoring my saved game, it ignored previously saved game, and half the plugins are not working.
